src/components/analyses/listing/Actions.js

```diff
@@ -49,16 +50,29 @@ const RelaunchButton = React.forwardRef((props, ref) => {
 });
 
 const GotoOutputFolderButton = React.forwardRef((props, ref) => {
-    const { baseId, className, onClick, href } = props;
+    const {
+        baseId,
+        className,
+        onClick,
+        href,
+        isTerminated,
+        setPendingTerminationDlgOpen,
+    } = props;
     const { t } = useTranslation("analyses");
     return (
         <IconButton
             size="small"
             id={build(baseId, ids.ICONS.OUTPUT, ids.BUTTON)}
             className={className}
-            href={href}
-            onClick={onClick}
-            ref={ref}
+            href={isTerminated ? href : null}
+            ref={isTerminated ? ref : null}
+            onClick={(event) => {
+                if (isTerminated) {
+                    onClick();
+                } else {
+                    setPendingTerminationDlgOpen(true);
```

Comment:
I would have used `stopPropagation` and `preventDefault` here, but then the row doesn't get selected, which means I don't have access to the analysis name. So I opted for disabling the `href` and `ref` instead